portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,7 +14,9 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
```

Comment:
In order to use Gson you have to add it to your maven package dependencies in **pom.xml**. 

See https://github.com/google/gson/blob/master/UserGuide.md#using-gson-with-maven

Add this to your **pom.xml** file inside of `<dependencies> ... </dependencies>`
```xml
<!--  Gson: Java to Json conversion -->
<dependency>
  <groupId>com.google.code.gson</groupId>
  <artifactId>gson</artifactId>
  <version>2.8.6</version>
  <scope>compile</scope>
</dependency>
```

Reply:
Ah I added it in, but I forgot to stage my pom changes.